spinflip/matrix.py

```diff
@@ -55,15 +61,30 @@ def visualizer(M, log=True,  text='mag', traceless = True, vmin=1E-15,vmax=1E-6,
             for m in np.arange(0,matrix_size):
                 xcoord = (n+1/6)*grid_subdivisions
                 ycoord = 1 - (m+1/2)*grid_subdivisions
-                ax.text(xcoord, ycoord, str(round(np.log10(np.abs(M[m,n])),0)), color='cyan', size=10)
+                if log == True:
+                    ax.text(xcoord, ycoord, str(round(np.log10(np.abs(M[m,n])),0)), color='cyan', size=10)
+                else:
+                    ax.text(xcoord, ycoord, str(round(np.abs(M[m,n]),2)), color='cyan', size=10)
                 
     elif text=='arg':
         for n in np.arange(0,matrix_size):
             for m in np.arange(0,matrix_size):
                 xcoord = (n)*grid_subdivisions
                 ycoord = 1 - (m+1/2)*grid_subdivisions
                 ax.text(xcoord, ycoord, str(round(np.real(M[m,n]/np.abs(M[m,n])), ndigits=1))+'+'+str(round(np.imag(M[m,n])/np.abs(M[m,n]), ndigits=2))+'i', color='cyan', size=9)
```

Comment:
Shouldn't the argument involve a call to arctan2? Also, why is one rounded to only one digit and the other is rounded to two digits?

Reply:
argument doesnt actually show the argument (in the sense of an angle) -- it returns a normalized complex number. If log = True I usually just want an order of magnitude and digits after the decimal just get in the way; if log = False I'm usually comparing specific numbers and want extra digits. this is just a visualization tool anyway, I doubt it will go in the paper

Reply:
Ah, I see. Would 'mag' or 'norm' be appropriate?